Bio/Entrez/__init__.py

```diff
@@ -46,8 +46,12 @@
     - email        Set the Entrez email parameter (default is not set).
     - tool         Set the Entrez tool parameter (default is ``biopython``).
     - api_key      Personal API key from NCBI. If not set, only 3 queries per
-                   seconds are allowed. 10 queries per seconds otherwise with a
+                   second are allowed. 10 queries per seconds otherwise with a
                    valid API key.
+    - max_tries    Configures how many times failed requests will be
+                   automatically retried on error (default is 3).
+    - sleep_between_tries   The delay, in seconds, before retrying a request on
+                            error (default is 15).
```

Comment:
My internet is flakey today so I don't have access to an RST preview, but I think the last three bullet points (including ``api_key``) need the continuation lines to have less indentation (like the next block of bullet points).

Reply:
You may well be right. The formatting here was actually already broken before I touched it - see https://biopython.org/DIST/docs/api/Bio.Entrez-module.html or https://biopython.org/docs/dev/api/Bio.Entrez.html.

It's on my TODO list to fix; I put it off yesterday because I was working on a machine with a broken `make` and so was failing to build the docs. Since the formatting was already broken I was considering fixing it a separate task from documenting the new variables, but feel free to hold off on merging this until I've had a chance to swing back and fix it all.

Reply:
I agree it was already broken. You might as well fix it here now that we've spotted it?

BTW, I normally use http://rst.ninjs.org/ for a quick copy-paste check, but looks to be offline - https://livesphinx.herokuapp.com/ is a working fork

Reply:
Fix pushed.

(Ironically the previous formatting rendered alright at https://livesphinx.herokuapp.com/, due to a difference in stylesheet.)